ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v1/aemform/aemform.html

```diff
@@ -40,8 +40,9 @@
 </script>
 <sly data-sly-use.clientLib="${'/libs/granite/sightly/templates/clientlib.html'}"/>
 <sly data-sly-test.useIframe="${form.useIframe != 'false'}">
+    <sly data-sly-test.formLocaleString="${form.LocaleString || 'en'}"/>
```

Comment:
Shouldn't it be form.locale || 'en' ?

Reply:
Oops.
Sorry, My bad.
I was referring to the AdaptiveForm class from foundation package instead of AEMForm class from core-components package

Yes this should be form.locale, will fix it right away

Reply:
Updated at re-created PR at https://github.com/adobe/aem-core-forms-components/pull/9
My pushes were not being added in this PR for some reason, and force pushes weren't registered either, so created a new PR